docs/src/pages/components/badge.mdx

```diff
@@ -1,2 +1,231 @@
+import { Link } from 'gatsby';
+import { BadgeWithParentExample } from '../../components/BadgeWithParentExample';
+
 # Badge
 
+The Badge component displays a dot or a circle with a count. It is normally used to indicate pending items (items in cart, unread messages, etc.)
+
+## Usage
```

Comment:
Heads up, I'm going to hit you with some formatting comments since I'm trying to keep all the docs consistent. Right now I'm staying away from the **Usage** title. Just give an explanation of the component at the top, usage is kind of implied once you start to give code examples.

Reply:
haha sounds good 👍